3.distributed-calculator/README.md

```diff
@@ -9,7 +9,90 @@ This sample shows method invocation and state persistent capabilities of Dapr th
 
 The front-end application consists of a server and a client written in [React](https://reactjs.org/). 
 Kudos to [ahfarmer](https://github.com/ahfarmer) whose [React calculator](https://github.com/ahfarmer/calculator) 
-sample was used for the client. The following architecture diagram illustrates the components that make up this sample: 
+sample was used for the client.
+
```

Comment:
The "Prerequisites" section should be here. It should include two subsections one for running locally and one for K8S.
For the running locally, seems like prerequisites include installing:
- go
- npm and node
- python3 and pip3
- dotnet core 

(links should be added to official releases of these tools/lang)